ambry-network/src/test/java/com.github.ambry.network/SSLBlockingChannelTest.java

```diff
@@ -116,7 +117,7 @@ public void testWrongPortConnection()
       channel.connect();
       fail("should have thrown!");
     } catch (IOException e) {
-      assertEquals(e.getMessage(), "Connection refused");
+      assertTrue(e.getMessage().contains("Connection refused"));
```

Comment:
It's probably a bad idea to be checking for exception messages in the first place.